### DISCUSSION TITLE
Welcome to GxEPD2 Discussions!

### DISCUSSION THREAD

**Discussion by ZinggJM** [ZinggJM/GxEPD2#69] [Announcements]
## 👋 Welcome!
  I enable using Discussions as a place to connect with other members of this community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  This is also an alternative to the Arduino Forum Displays, where I will be less present.
  
 To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kspearAus** [comment]
Hello ZinggJM!
I am an old retired IT professional who discovered your display support in my COVID hobby microcontroller based projects.  Your work helped me use a 7.8" e-ink display for a weather center when they first became available.  Thank you for your extensive contributions.  My hobby world is brighter because of it.

**Comment by desiFish** [comment]
Hey,
I am a hobbyist. Recently left my job, hence a lot of time in my hand. I am 27. Yes, I am looking for a job, but the process is boring hence I am full on making arduino stuff. I have two projects using E-Paper display, one of them uses your library and it made using it a lot easier. But your library doesn't support the other one, hence I use it with the stock library.
I just saw your post and I thought I should write something. Kudos to ZinggJM.

**Reply by kspearAus** [reply]
Ouch!  Best of luck with your job search.

**Reply by desiFish** [reply]
Thank You :)

**Comment by ANDRE888** [comment]
Thank you for your GxEPD2 project. Thanks to it, I built a home weather station that displays hourly and weekly weather forecasts. I have been using it every day for over 4 years.

**Comment by ThB4-UdeS** [comment]


**Reply by ZinggJM** [reply]
Please remove your post and create a new topic under Q&A.
